docs/documentation/publishing-on-govuk-paas.md

```diff
@@ -0,0 +1,130 @@
+# Publishing on the web (GOV.UK PaaS)
+
+[GOV.UK PaaS](https://www.cloud.service.gov.uk/) is another way to get your prototype online. It’s simple and fast to deploy new versions as you work, learning <code>git</code> isn't essential, and the platform is accredited to 'Official'.
+
+Once your prototype is on PaaS, other people will be able to access and try your prototype from their own computers or mobile devices.
+
+> We'll show how to secure your prototype in step 6 but *DO NOT* enter real user data in to prototypes hosted on PaaS. If your prototype stores or collects user data, talk to a security professional about appropriate security steps you and your developer colleagues must take.
+
+## 1) Sign up for an account
+You need to be able to log in to your account each time you want to publish a new version of a prototype.
+
+[Create an account](https://www.cloud.service.gov.uk/request-trial.html) and return to this page, or move on to step 2 if you already have one.
+
+## 2) Install the PaaS/Cloud Foundry command line tools
+
+GOV.UK PaaS is built on top of an open source technology called Cloud Foundry. You need to install a command line interface (<abbr>CLI</abbr>) tool to interact with it through a terminal window.
+
+There are a couple of ways to do the installation.
+
+* You can [get the install package (.pkg)](https://github.com/cloudfoundry/cli#installers-and-compressed-binaries) from the Cloud foundry github page. Once downloaded double click to start the installation.
+* Or if you [use a package manager](https://github.com/cloudfoundry/cli#installing-using-a-package-manager) the Cloud Foundry CLI can be installed that way
+
+![Installer downloads are quite a long way down the github page](/public/images/docs/cloudfoundry_cli_installers.png)
+
+>Because PaaS is built on top of Cloud Foundry, the most common command is `cf` - asking the underlying technology to do things with your code on a remote server.
+
+After following your preferred method, check that it's installed correctly by opening a terminal window and running
+
+```
+cf -v
+```
+
+You should get a message like this, confirming the installed version:
+
+```
+cf version X.X.X...
+```
+
+For more information check the [PaaS setup docs](https://docs.cloud.service.gov.uk/#quick-setup-guide)
+
+## 3) Login to GOV.UK PaaS
+
+Log in on the commandline by typing this
+
+```
+cf login -a api.cloud.service.gov.uk -u USERNAME
+```
+
+Where **USERNAME** is the email address your account was created with.
+
+You’ll then be prompted to enter your password.
+
+## 4) Choose name for prototype
+Go to the home folder of your prototype and see if the `manifest.yml` file exists. For a freshly downloaded prototype kit it should look similar to this:
```

Comment:
@joelanman - would it be worth including a manifest.yml file by default in the kit?